flink-connector-mysql-cdc/src/main/java/com/ververica/cdc/connectors/mysql/source/utils/LogUtils.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.ververica.cdc.connectors.mysql.source.utils;
+
+import java.lang.reflect.Method;
+
+/** Utilities for log. */
+public class LogUtils {
+    /**
+     * Set loggers' level to error.
+     *
+     * @param loggersName the loggers' name
+     */
+    public static void setLogLevelToError(String... loggersName) throws Exception {
+        Class<?> logConfiguratorClz =
+                Class.forName("org.apache.logging.log4j.core.config.Configurator");
+        Class<?> logLevelClz = Class.forName("org.apache.logging.log4j.Level");
```

Comment:
This is very hack. We can't assume users use log4j as the implementation of slf4j. 
That means there may throw Class Not Found exception when using logback or others.